src/sentry/models/apitoken.py

```diff
@@ -53,6 +55,15 @@ class Meta:
     def __str__(self):
         return force_str(self.token)
 
+    def save(self, **kwargs: Any) -> None:
+        # when a new ApiToken is created we take the last four characters of the token
+        # and save them in the `token_last_characters` field so users can identify
+        # tokens in the UI where they're mostly obfuscated
+        token_last_characters = self.token[-4:]
+        self.token_last_characters = token_last_characters
```

Comment:
Is there a reason we want to store this in the database, vs just obfuscating what we return from the api?

Reply:
The eventual goal is to not store these tokens in plaintext, but rather as a SHA256 hash.